source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -76,6 +74,17 @@
 
 #define QUIC_PREDICT_FALSE_IMPL(x) ABSL_PREDICT_FALSE(x)
 
+#define CHECK(condition)                                                                           \
```

Comment:
Can you move this CHECK to before the #ifdef at line 56?

Reply:
done